mqtt-ds/src/main/java/org/apache/rocketmq/mqtt/ds/notify/NotifyManager.java

```diff
@@ -281,7 +280,7 @@ protected void sendEventRetryMsg(Set<MessageEvent> messageEvents, int delayLevel
         }
         Message message = new Message();
         message.setTopic(serviceConf.getEventNotifyRetryTopic());
-        message.setBody(JSONObject.toJSONString(messageEvents).getBytes(StandardCharsets.UTF_8));
```

Comment:
Would you like to add some unit tests to verify these changes？

Reply:
done